### PR TITLE
[SRVCOM-3134] Update API version for connecting-channel-sinks

### PR DESCRIPTION
**Pull request by ochromy**
Version(s):
serverless-docs-1.33+

Issue:
https://issues.redhat.com/browse/SRVCOM-3134

Link to docs preview:
https://78923--ocpdocs-pr.netlify.app/openshift-serverless/latest/eventing/channels/connecting-channels-sinks.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/serverless-creating-subscriptions-kn.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ $ kn subscription create <subscription_name> \
   --sink <sink_prefix>:<sink_name> \ <2>
   --sink-dead-letter <sink_prefix>:<sink_name> <3>
 ----
-<1> `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default `InMemoryChannel` channel that is backed by the `Channel` custom resource, you must prefix the channel name with the `<group:version:kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1beta1:KafkaChannel` for an Apache Kafka backed channel.
+<1> `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default `InMemoryChannel` channel that is backed by the `Channel` custom resource, you must prefix the channel name with the `<group:version:kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1:KafkaChannel` for an Apache Kafka backed channel.
```
Comment (maschmid) [inline modules/serverless-creating-subscriptions-kn.adoc:28]: While `Channel` would be `messaging.knative.dev:v1:Channel` , the `KafkaChannel` is still just `v1beta1`, so this line should stay as it was.

**Review by maschmid — CHANGES_REQUESTED** [review]

**Comment by maschmid** [comment]
Hi, the changes in [modules/serverless-creating-subscriptions-yaml.adoc](https://github.com/openshift/openshift-docs/pull/78923/files#diff-ace56381e9f18c030d18eb20534d354033a62fc7797f1c77471d35148dffd3d3) are OK, but the KafkaChannel version change in https://github.com/openshift/openshift-docs/pull/78923/files#diff-d97b7362ba6c02ad7f15ebfb0c940abc44ec2a82629288974758abc58a7df03f  should be reverted.

**Review by snarayan-redhat — COMMENTED** [review]
LGTM!

**Pull request merged**